src/global-styles.js

```diff
@@ -94,6 +94,11 @@ a.pf-c-breadcrumb__item {
     z-index: 1;
   }
 }
+
+.pf-c-modal-box__body {
+  overflow-y: hidden !important;
```

Comment:
This will cause an issue on small devices or when the form has many fields (a big list of configuration when ordering job template).

Users won't be able to scroll down to the overflowing content.

![screenshot-ci cloud redhat com-2020 05 27-08_41_54](https://user-images.githubusercontent.com/22619452/82986418-13bac580-9ff6-11ea-9308-c56dbbdb4519.png)

The solution here is directly tied to the PF modal HTML hierarchy. And I am not sure if we can simply fix it by changing CSS until they give us full control over how we can build these dialogs.